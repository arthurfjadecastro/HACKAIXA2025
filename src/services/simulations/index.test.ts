import * as simulationsIndex from './index';

describe('Simulations Services Index', () => {
  it('should export simulations modules', () => {
    expect(simulationsIndex).toBeDefined();
    expect(typeof simulationsIndex).toBe('object');
  });

  it('should re-export API and calculator modules', () => {
    // Verifica se mÃ³dulos foram exportados sem erros
    expect(() => {
      const exported = { ...simulationsIndex };
      expect(exported).toBeDefined();
    }).not.toThrow();
  });

  it('should match snapshot', () => {
    expect(Object.keys(simulationsIndex).sort()).toMatchSnapshot();
  });
});
