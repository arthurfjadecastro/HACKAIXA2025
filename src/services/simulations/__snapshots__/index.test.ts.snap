// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simulations Services Index should match snapshot 1`] = `
[
  "calculateLoanSchedule",
  "convertMonthlyToYearlyRate",
  "convertYearlyToMonthlyRate",
  "simulationsApi",
]
`;
